README.md

```diff
@@ -5,167 +5,145 @@
 [![Build Status][1]][2]
 
 ## Overview
-Heptio Ark is a utility for managing disaster recovery, specifically for your [Kubernetes][14] cluster resources and persistent volumes. It provides a simple, configurable, and operationally robust way to back up and restore applications and PVs from a series of checkpoints. This allows you to better automate in the following scenarios:
 
-* **Disaster recovery** with reduced TTR (time to respond), in the case of:
-    * Infrastructure loss
-    * Data corruption
-    * Service outages
+Ark gives you tools to back up and restore your Kubernetes cluster resources and persistent volumes. Ark lets you:
 
-* **Cross-cloud-provider migration** for Kubernetes API objects (cross-cloud-provider migration of persistent volume snapshots not yet supported)
+* Take backups of your cluster and restore in case of loss.
+* Copy cluster resources across cloud providers. NOTE: Cloud volume migrations are not yet supported.
+* Replicate your production environment for dev and testing environments.
 
-* **Dev and testing environment setup (+ CI)**, via replication of prod environment
+Ark consists of:
 
-More concretely, Heptio Ark combines an in-cluster service with a CLI that allows you to record both:
-1. *Configurable subsets of Kubernetes API objects* -- as tarballs stored in object storage
-2. *Disk snapshots of Persistent Volumes* -- via the cloud provider APIs
+* A server that runs on your cluster
+* A CLI
 
-Heptio Ark currently supports the [AWS][15], [GCP][16], and [Azure][17] cloud provider platforms.
+## Getting started
 
-## Quickstart
+The following example sets up the Ark server and client, then backs up and restores a sample application.
 
-This guide gets Ark up and running on your cluster, and goes through an example using the following:
-* **Minio, an S3-compatible storage service** that runs locally on your cluster. This is the storage service where backup files are uploaded. *Note that Ark is intended to run on a cloud provider--we are using Minio here to keep the example convenient and self-contained.*
+For simplicity, the example uses Minio, an S3-compatible storage service that runs locally on your cluster. See [Set up Ark with your cloud provider][3] for how to run on a cloud provider. 
 
-* **A sample nginx app** under the `nginx-example` namespace, used to demonstrate Ark's backup and restore functionality.
+### Prerequisites
 
-Note that this example *does not* include a demonstration of PV disk snapshots, because that feature requires integration with a cloud provider API. For snapshotting examples and instructions specific to AWS, GCP, and Azure, see [Cloud Provider Specifics][23].
+* Access to a Kubernetes cluster, version 1.7 or later. Version 1.7.5 is required to run `ark backup delete`.
```

Comment:
Should it be "Version 1.7.5 *or higher* is required to run `ark backup delete`."? I assume it doesn't work on exclusively 1.7.5.

Reply:
Correct